__tests__/api/index.js

```diff
@@ -6,6 +6,7 @@ import {
   ERR_MODULE_NOT_FOUND,
   ERR_MODULE_DISABLED,
   ERR_MODULE_REMOVE_FAILED,
+  ERR_MODULE_SEARCH_FAILED,
```

Comment:
Woops, looks like I left that out as well! Will get to this piece tonight otherwise feel free to send a quick PR. 👍